query-engine/js-connectors/src/proxy.rs

```diff
@@ -106,37 +168,143 @@ pub struct Query {
     pub args: Vec<serde_json::Value>,
 }
 
+fn js_planetscale_value_to_quaint(json_value: serde_json::Value, column_type: ColumnType) -> QuaintValue<'static> {
+    // Note(jkomyno): I suggest we avoid so many different panic messages.
+    // They bloat the compiled code, making it heavier. I suggest we consider the type information
+    // received from the JS connector safe, and eventually just let the code panic without a custom message here.
+    // We can later infer which column type is not properly supported from the panic message's line and column anyway.
```

Comment:
Unnecessary and premature optimization. Each of the panic! formatting string in the code account for a total of less than 1KB.

Reply:
Totally agree, hence why I didn't perform such an optimisation. But leaving a note here will let us ring some memory bells when we glance back at the code later on.

Reply:
I believe you better remove yourself from the source code comments, otherwise you might risk this to be interpreted as you own that optimisation. Or that someone must ask you about it, while the source code and their comments should be self explanatory. 

Besides, `git annotate` also provides a functionality for everyone to understand who left the comment.

Also the comment is not based on facts. Binary bloat is not binary as there is bloat or there isn't. The relevance lies on how much bloat is acceptable. And in this, given the source code, is not evident it might ever be a problem, not certainly until measured. 

An alternative comment might be.

```suggestion
    // May be worth to revisit how much bloat different static strings add to the binary, and in case necessary come up with a restricted set of panic messages.
```